docs/config-file/v2.rst

```diff
@@ -120,6 +120,11 @@ The Python version (this depends on :ref:`config-file/v2:build.image`).
 :Type: ``number``
 :Default: ``3``
 
+.. warning::
+
+  If you are using a :ref:`Conda <config-file/v2:conda>` environment to manage
+  the build, this setting will not have any effect, as the Python version is managed by Conda.
```

Comment:
@stsewd what about making these notes as validation errors when the build is triggered? I feel that it only adds confusion to users the ability to use both at the same time and do not receive any error when building?

Reply:
Yeah, we can raise an error in addition to just put it in the docs

Reply:
@humitos I think your question is answered in the first line of the description of  #5388 

> We should mention the restrictions of using conda, a better approach would be to raise an exception in the validation process, but that would break some projects, so that's not possible now.

Reply:
https://github.com/readthedocs/readthedocs.org/issues/6569

Reply:
@Parth1811 thanks for pointing that, didn't remember I wrote that :smile: